zuko-ui/src/pages/index.js

```diff
@@ -1,7 +1,10 @@
+import LayoutWrapper from "../components/LayoutWrapper/LayoutWrapper";
+import OurValues from "../components/OurValues/OurValues";
 import TestComponent from "../components/TestComponent/TestComponent";
 
 export default function HomePage() {
     return <div>
         <TestComponent testProperty="Example component"/>
+        <LayoutWrapper children={<OurValues/>} />
```

Comment:
You shouldn't use LayoutWrapper component because that component is already wrapping HomePage component.

Also, this component <OurValues /> should be displayed on the "About us" page